runtime/lua/vim/_watch.lua

```diff
@@ -198,7 +199,7 @@ end
 --- directory at path.
 ---
 ---@param path (string) The path to watch. Must refer to a directory.
----@param opts (table|nil) Additional options
+---@param opts watch.PollOpts? (table|nil) Additional options
```

Comment:
Isn't this redundant?
```suggestion
---@param opts watch.PollOpts? Additional options
```

Reply:
No. it's to control the doc-gen. Which this `(table|nil)` will be documented.

Reply:
In that case why keep the `?`?

Reply:
Because that is a part of the type.

Reply:
Ah, we don't generate docs from this file so it is redundant.

Reply:
> Ah, we don't generate docs from this file so it is redundant.

So does this mean that you agree with me on removing the `(table|nil)`?

Reply:
Yes, as like you said it is redundant. I assumed because the docstring was complete that we exported docs for this.

Reply:
Then why haven't you removed the redundancy?